service/test/integration1_test.go

```diff
@@ -945,6 +945,14 @@ func Test1Disasm(t *testing.T) {
 			if strings.HasPrefix(curinstr.Text, "call") || strings.HasPrefix(curinstr.Text, "CALL") {
 				t.Logf("call: %v", curinstr)
 				if curinstr.DestLoc == nil || curinstr.DestLoc.Function == nil {
+					// On 386 linux when pie, maybe meet  "call __x86.get_pc_thunk." first but not find any symbol name or file name.
```

Comment:
I think that instead of doing this we should skip the test for 386. Before the for loop:

```
if runtime.GOOS == "386" {
    // Skip the rest of the test because on intel 386 the compiler will insert calls to __x86.get_pc_thunk which do not have DIE and we can't resolve.
    return
}
```

Reply:
1. when nopie for 386 linux, it is ok

2. For user, `step` can't enter `__x86.get_pc_thunk`, but `step instruction` can.  We maybe need to keep this testcase to verify this logic.

Reply:
Need to discuss

Reply:
Ok then `if runtime.GOOS == "386" && buildMode == "pie"`

Reply:
Done

Reply:
What I meant is to add:

```
if runtime.GOOS == "386" && buildMode == "pie" {
    // Skip the rest of the test because on intel 386 with PIE build mode the compiler will insert calls to __x86.get_pc_thunk which do not have DIEs and we can't resolve.
    return
}
```

before the for loop.

Reply:
Sorry, has done.